src/components/card/Card.stories.js

```diff
@@ -410,3 +410,18 @@ Grid.args = {
   grid_type: 'threecol--33-34-33',
   record_count: 3,
 }
+
+export const DateHighlight = GridTemplate.bind({})
```

Comment:
```suggestion
export const MediaDate = GridTemplate.bind({})
```
I don't think we should call this a highlight since it is confusing in the context of other places we use that term.

Also, should this be just extending one of the single card classes? Or do we just find the grid presentation to be more useful and maybe we should get rid of the single card classes?

Reply:
Truthfully I don't find much value in looking at the cards in the singles. I dont get a clear picture of the intended look, and it is not possible to correctly show a media left or right example from what I've seen.

I will commit the change to the name though!